extensions/positron-r/syntaxes/r.tmGrammar.src.json

```
@@ -431,7 +431,7 @@
 			]
 		},
 		"roxygen": {
-			"begin": "(^\\s*#+')",
+			"begin": "(^\\s*#+(?:'|*))",
```

Do you need to escape the `*` here?

```suggestion
			"begin": "(^\\s*#+(?:'|\*))",
```